app/controllers/auth.rb

```diff
@@ -8,7 +8,18 @@ class SlowFoodApp
   end
 
   get '/signup' do
+    erb :signup
+  end
 
+  post '/signup' do
+    @new_user = User.new(name: params['user']['username'], password: params['user']['password'], password_confirmation: params['user']['passwordconfirmation'])
+    if @new_user.save
+      flash[:success] = 'Your user has been created'
+      redirect '/'
+    else
+      flash[:error] = @new_user.errors.full_messages
```

Comment:
Will this display a string or an array? Take a look using `show me the page`

Reply:
Fixed